styles/Canonical/012-Date-format.yml

```diff
@@ -0,0 +1,15 @@
+extends: existence
+message: Use the '1 January 1970' format instead of '%s'
+scope: raw
```

Comment:
I wonder about the raw scop here, mainly because I can see these date formats turning up in other artifacts we have no control over (filenams, links, etc). We really only care about dates in body text i think so maybe `summary`would be sufficient?

Reply:
I agree, we shouldn't check code blocks or file names.